spec/unit/views/components/table_for_spec.rb

```diff
@@ -99,6 +99,54 @@
       end
     end
 
+    context "when creating an id column" do
+      let(:table) do
+        render_arbre_component assigns, helpers do
+          table_for(collection, resource_class: Post) do
+            id_column
+          end
+        end
+      end
+
+      it "should create a table header based on the primary key" do
+        expect(table.find_by_tag("th").first.content).to eq "Id"
+      end
+
+      it "should create a table row for each element in the collection" do
+        expect(table.find_by_tag("tr").size).to eq 4 # 1 for head, 3 for rows
+      end
+
+      [1, 2, 4].each_with_index do |content, index|
```

Comment:
Could you change this to be an array of strings and remove the `to_s` below? I think it makes it more clear that these mean content and not indexes.